examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -86,7 +86,7 @@ repo_name = "terraform-aws-ecs-codepipeline"
 
 branch = "master"
 
-build_image = "aws/codebuild/docker:17.09.0"
```

Comment:
It might be worth updating the reference to this in the example vars description too @Roondel